components/pixel/schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "title": "Pixel Component Definition",
+  "description": "Addressable Pixel WipperSnapper component for use in Adafruit IO",
+  "type": "object",
+  "required": [ "displayName", "pixelType"],
+  "properties": {
+    "displayName": {
+      "description": "The human-friendly name of this component.",
+      "type": "string",
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "pixelType": {
+        "description": "The type of addressable pixel. 0 is unspecified, 1 is WS2812 (NeoPixel), 2 is APA201 (DotStar).",
+        "type": "number",
+        "minLength": 0,
+        "maxLength": 2
+    },
+    "defaultNumPixels": {
+        "description": "If present, the component form will allow the user to set the number of pixels in the strand, with this value as the default.",
+        "type": "number",
+        "minLength": 0,
+        "maxLength": 156
+    },
+    "defaultPixelBrightness": {
+        "description": "If present, the component form will allow the user to set the pixel strand's brightness, with this value as the default.",
+        "type": "number",
+        "minLength": 0,
+        "maxLength": 255
+    },
```

Comment:
These are the wrong modifier for "number" type:
- `minLength` should be `minimum`
- `maxLength` should be `maximum`

...I think. Here's the reference: https://ajv.js.org/json-schema.html#keywords-for-numbers

Reply:
Thanks, changed in https://github.com/adafruit/Wippersnapper_Components/pull/44/commits/2b275e6fd229ec4e81d4c4b9862e14cc844408d8